commcare_connect/opportunity/views.py

```diff
@@ -228,11 +210,15 @@ def get_success_url(self):
 
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
-        kwargs["domains"] = get_domains_for_user(self.request.user)
         kwargs["user"] = self.request.user
         kwargs["org_slug"] = self.request.org.slug
         return kwargs
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["api_key_form"] = HQApiKeyCreateForm(auto_id="api_key_form_id_for_%s")
```

Comment:
Is the `%s` literal or is something being interpolated there?

Reply:
The %s is needed for automatic id generation of the field. There were two forms on the same page with similar fields that caused issues so this ensures that the ids generated are unique due to the prefix added here.